packages/@azure/servicebus/data-plane/test/stress/fixedNumberOfClientsMultipleQueues.ts

```diff
@@ -0,0 +1,62 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
+
+const connectionString = "";
+
+const testDurationInMilliseconds = 60000 * 5 * 12 * 24 * 7; // 1 week
+
+const numOfClients = 10;
+
+let msgId = 1;
+
+let snapshotIntervalID: any;
+
+let isJobDone = false;
+
+async function main(): Promise<void> {
+  snapshotIntervalID = setInterval(snapshot, 5000); // Every 5 seconds
+  setTimeout(() => {
+    isJobDone = true;
+  }, testDurationInMilliseconds);
+
+  sendReceiveMessages();
+}
+
+async function sendReceiveMessages(): Promise<void> {
+  const ns = ServiceBusClient.createFromConnectionString(connectionString);
+
+  const clients = [];
+  for (let i = 0; i < numOfClients; i++) {
+    clients[i] = ns.createQueueClient(`t1-queue-new-${i + 1}`);
+  }
+
+  try {
+    while (!isJobDone) {
+      for (let i = 0; i < numOfClients; i++) {
+        const sender = clients[i].createSender();
```

Comment:
This would be contemplating on what the intent of the tests would be which we had clarified in meeting on 3/28. 
Are we looking to re-define the P0 scenarios as such, or simply ensuring that close is invoked on test exit?

Reply:
If we don't call close, that will leak the sender/receiver. What we want to test real are user scenarios, which means we should not leak objects if we can close them.

I don't think this is re-defining the P0 scenarios -- just making sure they are correctly implemented to not leak resources -- this will make sure that the issues we find are not about leaking objects.